Makefile

```diff
@@ -24,8 +24,14 @@ composer-install-nodev:
 pot:
 	vendor/bin/wp i18n make-pot . languages/onoffice.pot --skip-js
 	sed -i -r "s/onOffice for WP-Websites \(dev\)/onOffice for WP-Websites/" languages/onoffice.pot
+
+concat-css:
+	@cat `ls -I ./third_party/slick/*\.css`  ./css/*\.css > ./css/build/oo-wp-plugin.css
+
+mkdirs:
+	@mkdir ./css/build/
```

Comment:
What do you think about not having the `build` sub-directory, but naming the new CSS file something like `all.css`?

Reply:
the `mkdirs` target could be dropped then. I would really like that.

Reply:
> What do you think about not having the `build` sub-directory, but naming the new CSS file something like `all.css`?
I have nothing against all.css, but we can't do without a subdirectory, because `./css/*\.css` searches for all CSS files in this folder and in another process all.css would be in there as well.

Reply:
And in another pull request I will summarize the JavaScript files, I think we will not be able to avoid mkdirs.

Reply:
Dealing with the Javascript in a second PR sounds good. I'm gonna need a little bit of time to think through the mkdir, though.

Reply:
Maybe I should rename 'build' into 'dist'?